hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/upgrade/UpgradeDowngrade.java

```diff
@@ -275,4 +281,98 @@ protected Pair<Map<ConfigProperty, String>, List<ConfigProperty>> downgrade(Hood
       throw new HoodieUpgradeDowngradeException(fromVersion.versionCode(), toVersion.versionCode(), false);
     }
   }
+
+  protected UpgradeHandler getUpgradeHandlerInstance(HoodieTableVersion fromVersion, HoodieTableVersion toVersion) {
+    if (fromVersion == HoodieTableVersion.ZERO && toVersion == HoodieTableVersion.ONE) {
+      return new ZeroToOneUpgradeHandler();
```

Comment:
is there any way we can reuse code with `UpgradeDowngrade#upgrade` and `UpgradeDowngrade.downgrade` ?

Reply:
I can try looking into this